docs/user-docs/parallel-orchestration.md

```
@@ -164,30 +169,30 @@ parallel:
 | `/gsd parallel merge` | Merge all completed milestones back to main |
 | `/gsd parallel merge M002` | Merge a specific milestone back to main |
 
-## Signal Lifecycle
+## Command Lifecycle
 
-The coordinator communicates with workers through signals:
+The coordinator communicates with workers through persisted runtime commands:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)